econml/tests/test_drlearner.py

```diff
@@ -1,712 +1,804 @@
-# Copyright (c) Microsoft Corporation. All rights reserved.

```

Comment:
Looks like maybe a problem with the line endings... See if you can keep your changes but revert to Unix-style line endings.

Reply:
when I committed, there was a conflict on this file because of the update on master. I used the web interface to solve and conflict and for some reason github identified the whole file has been changed...I compared the two file adding my changes without changing anything for the current file on master. Don't know exactly how to recover the previous version, please help here if we should go back to the previous master version to make sure everything is unchanged and add my stuff again.

Reply:
I think if you make any trivial change to this file (e.g. add or remove a blank line at the end of the file) on your own machine and commit it then all of the line endings will be fixed automatically.